### DISCUSSION TITLE
jupyter kernel when plotting geodetic  line with shapely 1.8.1post1 but works OK with with version 1.8.0

### DISCUSSION THREAD

**Discussion by paulathomas** [shapely/shapely#1338] [Q&A]
when I try to plot a geodetic line with cartopy 0.20.2 and matplotlib 3.5.1 (using the 'transform' parameter in pyplot.plot) the kernel restarts with no output if I use shapely 1.8.1post1 but works as expected with 1.8.0.

I am using wsl2 with ubuntu 21.10 

If any more information is needed please ask.

**Comment by jorisvandenbossche** [comment]
You will need to give more details on how you installed those libraries. 
My first guess would be that the update of the GEOS version in the shapely 1.8.1 wheels is somehow interfering with cartopy (which also uses GEOS). 

Can you also see if you have output in the console where you started the notebook? (that might show some crash error traceback when a kernel dies / restarts)

**Reply by paulathomas** [reply]
Output on jupyter lab console with 1.8.1.post1:

`[I 2022-03-05 10:45:02.930 ServerApp] Saving file at /python/cartopy/Random Notes.ipynb
[I 2022-03-05 10:45:31.982 ServerApp] AsyncIOLoopKernelRestarter: restarting kernel (1/5), keep random ports
[W 2022-03-05 10:45:31.982 ServerApp] kernel 8cc01c62-8626-4048-9c44-571da45b89e1 restarted
[W 2022-03-05 10:45:31.983 ServerApp] kernel 8cc01c62-8626-4048-9c44-571da45b89e1 restarted
[I 2022-03-05 10:45:32.016 ServerApp] Starting buffering for 8cc01c62-8626-4048-9c44-571da45b89e1:0c34935e-e1d4-4e17-bc11-c1ef61091b0c
[I 2022-03-05 10:47:03.112 ServerApp] Saving file at /python/cartopy/Random Notes.ipynb
`

Hope this helps! As far as I can tell I  only have one copy of GEOS and it is version 3.9.0.1

**Reply by jorisvandenbossche** [reply]
How did you install Shapely?

**Reply by paulathomas** [reply]
Using pip - I don't use virtual environments,, OK I probably should but I don't.

**Reply by jorisvandenbossche** [reply]
If you use plain `pip install shapely`, the binary wheel includes its own version of GEOS, which is GEOS 3.10.2 for Shapely 1.8.1. If you already have a GEOS installed and you want to use that, you need to install shapely from source: `pip install shapely --no-binary`. See https://shapely.readthedocs.io/en/latest/project.html#source-distributions

**Reply by paulathomas** [reply]
Thanks  - can I ask why you pull in the other version of GEOS - specifically will any features not work if I use the earlier one?
